### PR TITLE
feat: add ability to run in web

### PR DESCRIPTION
**Pull request by benfdking**
Hello, 

Hope you don't mind me making this contribution. This adds ability for the vscode extension to run in the web browser such as on `github.dev` or `vscode.dev`.

To test run 'yarn dev:web'. This should open a web version of vscode, in there create a .sql file and then write a sql command and save.

I am not 100% sure this will work but definitely worked for me locally.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -56,11 +56,13 @@
     "onCommand:prettier-sql-vscode.format-selection"
   ],
   "main": "./out/extension.js",
+  "browser": "./out/extension.js",
```
Comment (nene) [inline package.json:59]: Do I understand correctly, that this is the only line that's actually needed to make this extension work in web?

The rest of the changes seem to be only about adding a commands to test it on web.

File: .vscode/settings.json [hunk 2]
```diff
@@ -7,5 +7,7 @@
     "out": true // set this to false to include "out" folder in search results
   },
   // Turn off tsc task auto detection since we have the necessary tasks as npm scripts
-  "typescript.tsc.autoDetect": "off"
+  "typescript.tsc.autoDetect": "off",
+  "editor.formatOnSave": true
```
Comment (nene) [inline .vscode/settings.json]: Why this change?

File: package.json [hunk 3]
```diff
@@ -69,7 +71,9 @@
     "pretest": "yarn run compile && yarn run lint",
     "test": "node ./out/test/runTest.js",
     "vsce:package": "vsce package",
-    "vsce:publish": "vsce publish"
+    "vsce:publish": "vsce publish",
+    "run-in-browser": "vscode-test-web --browserType=chromium --open-devtools --extensionDevelopmentPath=. .",
+    "dev:web": "concurrently \"yarn watch-web\" \"pnpm run-in-browser\""
```
Comment (nene) [inline package.json]: I would prefer only adding a single new command: `run-in-browser` to make it possible to test the extension in web-mode.

I don't really see a reason to add support for developing the extension in web-mode.

This would also eliminate the `concurrently` dependency.

File: .gitignore [hunk 4]
```diff
@@ -3,3 +3,6 @@ out/
 .vscode-test/
 
 *.vsix
+
+.idea/
```
Comment (nene) [inline .gitignore]: Why the `.idea` directory?

Are you developing this extension in IntelliJ?

**Review by nene — CHANGES_REQUESTED** [review]
Thanks for the pull request.

I think it's a nice addition, but I suggest trimming it down to bare minimum of changes needed to make it work for the web.

**Comment by benfdking** [comment]
Made some more minimal edits. Let me know if that works for you. 

Cheers for the quick review.

File: package.json (hunk 1 above)
Comment (benfdking) [reply package.json:59]: Yes pretty much. I think.

**Review by nene — APPROVED** [review]
Looks mostly good to me.

Though you should have left the `.vscode-test-web` into the `.gitignore` file.

However, to get this change actually into the release, we'll need to wait for @inferrinizzard ...and he hasn't really been around lately :(

**Comment by inferrinizzard** [comment]
I can make a release once this PR is merged ✅

**Comment by benfdking** [comment]
I just reintroduce the .vscode-test-web into the .gitignore file.

**Comment by benfdking** [comment]
Woo 🎉  @inferrinizzard thank you!

**Review by nene — APPROVED** [review]

**Comment by inferrinizzard** [comment]
Thanks for the quick work, @benfdking !
